zmsadmin/custom_scss/admin.scss

```
@@ -85,4 +85,4 @@ $fa-font-path: "../node_modules/@fortawesome/fontawesome-free/webfonts";
 @import 'bo-layout-admin-scss/themes/zms/form.scss';
 
 // Inserting CSS styles for accessibility
-@import 'bo-layout-admin-scss/themes/zms/accessibility.scss';
+@import 'bo-layout-admin-scss/themes/zms/accessibility.scss';
```

Stylelint (17.10.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)